TST: new appveyor test failure, test_arima111_predict_exog_2127

no fail
https://ci.appveyor.com/project/josef-pkt/statsmodels/build/1.0.2306/job/lkw1dx062p8lt0cm
failure:
https://ci.appveyor.com/project/josef-pkt/statsmodels/build/1.0.2313/job/747alvv1ffxt8ka9

this is for a PR
but now master also has this failure without any changes from the PR
(master has also a commit for ARMA start_params that would affect this, however the failure showed up before that merge)

Note the failing unit test mentions that it is a regression test, so there could be just some slightly different convergence, and the test tolerance is tighter than the convergence criterion allow. It looks like it would still agree at 5 decimals.

```
================================== FAILURES ===================================
_______________________ test_arima111_predict_exog_2127 _______________________
    def test_arima111_predict_exog_2127():
        # regression test for issue #2127
        ef =  [ 0.03005,  0.03917,  0.02828,  0.03644,  0.03379,  0.02744,
                0.03343,  0.02621,  0.0305 ,  0.02455,  0.03261,  0.03507,
                0.02734,  0.05373,  0.02677,  0.03443,  0.03331,  0.02741,
                0.03709,  0.02113,  0.03343,  0.02011,  0.03675,  0.03077,
                0.02201,  0.04844,  0.05518,  0.03765,  0.05433,  0.03049,
                0.04829,  0.02936,  0.04421,  0.02457,  0.04007,  0.03009,
                0.04504,  0.05041,  0.03651,  0.02719,  0.04383,  0.02887,
                0.0344 ,  0.03348,  0.02364,  0.03496,  0.02549,  0.03284,
                0.03523,  0.02579,  0.0308 ,  0.01784,  0.03237,  0.02078,
                0.03508,  0.03062,  0.02006,  0.02341,  0.02223,  0.03145,
                0.03081,  0.0252 ,  0.02683,  0.0172 ,  0.02225,  0.01579,
                0.02237,  0.02295,  0.0183 ,  0.02356,  0.02051,  0.02932,
                0.03025,  0.0239 ,  0.02635,  0.01863,  0.02994,  0.01762,
                0.02837,  0.02421,  0.01951,  0.02149,  0.02079,  0.02528,
                0.02575,  0.01634,  0.02563,  0.01719,  0.02915,  0.01724,
                0.02804,  0.0275 ,  0.02099,  0.02522,  0.02422,  0.03254,
                0.02095,  0.03241,  0.01867,  0.03998,  0.02212,  0.03034,
                0.03419,  0.01866,  0.02623,  0.02052]
        ue =  [  4.9,   5. ,   5. ,   5. ,   4.9,   4.7,   4.8,   4.7,   4.7,
                 4.6,   4.6,   4.7,   4.7,   4.5,   4.4,   4.5,   4.4,   4.6,
                 4.5,   4.4,   4.5,   4.4,   4.6,   4.7,   4.6,   4.7,   4.7,
                 4.7,   5. ,   5. ,   4.9,   5.1,   5. ,   5.4,   5.6,   5.8,
                 6.1,   6.1,   6.5,   6.8,   7.3,   7.8,   8.3,   8.7,   9. ,
                 9.4,   9.5,   9.5,   9.6,   9.8,  10. ,   9.9,   9.9,   9.7,
                 9.8,   9.9,   9.9,   9.6,   9.4,   9.5,   9.5,   9.5,   9.5,
                 9.8,   9.4,   9.1,   9. ,   9. ,   9.1,   9. ,   9.1,   9. ,
                 9. ,   9. ,   8.8,   8.6,   8.5,   8.2,   8.3,   8.2,   8.2,
                 8.2,   8.2,   8.2,   8.1,   7.8,   7.8,   7.8,   7.9,   7.9,
                 7.7,   7.5,   7.5,   7.5,   7.5,   7.3,   7.2,   7.2,   7.2,
                 7. ,   6.7,   6.6,   6.7,   6.7,   6.3,   6.3]
    
        # rescaling results in convergence failure
        #model = sm.tsa.ARIMA(np.array(ef)*100, (1,1,1), exog=ue)
        model = ARIMA(ef, (1,1,1), exog=ue)
        res = model.fit(transparams=False, iprint=0, disp=0)
    
        predicts = res.predict(start=len(ef), end = len(ef)+10,
                               exog=ue[-11:], typ = 'levels')
    
        # regression test, not verified numbers
        # if exog=ue in predict, which values are used ?
        predicts_res = np.array(
              [ 0.02612291,  0.02361929,  0.024966  ,  0.02448193,  0.0248772 ,
                0.0248762 ,  0.02506319,  0.02516542,  0.02531214,  0.02544654,
                0.02559099,  0.02550931])
    
        # if exog=ue[-11:] in predict
        predicts_res = np.array(
              [ 0.02591112,  0.02321336,  0.02436593,  0.02368773,  0.02389767,
                0.02372018,  0.02374833,  0.02367407,  0.0236443 ,  0.02362868,
                0.02362312])
    
>       assert_allclose(predicts, predicts_res, atol=1e-6)
statsmodels\tsa\tests\test_arima.py:2113: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
c:\py\lib\site-packages\numpy\testing\utils.py:1297: in assert_allclose
    verbose=verbose, header=header)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
comparison = <function compare at 0x000000002562F898>
x = array([ 0.02590947,  0.02321157,  0.02436523,  0.02368662,  0.0238974 ,
      ...72013,  0.02374899,  0.02367515,  0.02364594,  0.02363098,
        0.02362612])
y = array([ 0.02591112,  0.02321336,  0.02436593,  0.02368773,  0.02389767,
      ...72018,  0.02374833,  0.02367407,  0.0236443 ,  0.02362868,
        0.02362312])
err_msg = '', verbose = True
header = 'Not equal to tolerance rtol=1e-07, atol=1e-06', precision = 6
    def assert_array_compare(comparison, x, y, err_msg='', verbose=True,
< ...>
>                   raise AssertionError(msg)
E                   AssertionError: 
E                   Not equal to tolerance rtol=1e-07, atol=1e-06
E                   
E                   (mismatch 100.0%)
E                    x: array([ 0.025909,  0.023212,  0.024365,  0.023687,  0.023897,  0.02372 ,
E                           0.023749,  0.023675,  0.023646,  0.023631,  0.023626])
E                    y: array([ 0.025911,  0.023213,  0.024366,  0.023688,  0.023898,  0.02372 ,
E                           0.023748,  0.023674,  0.023644,  0.023629,  0.023623])
c:\py\lib\site-packages\numpy\testing\utils.py:665: AssertionError
============================== warnings summary ===============================
statsmodels/datasets/tests/test_data.py::TestDatasets::()::test_all_datasets
```

--- a/statsmodels/tsa/tests/test_arima.py
+++ b/statsmodels/tsa/tests/test_arima.py
@@ -2110,7 +2110,7 @@ def test_arima111_predict_exog_2127():
             0.02372018,  0.02374833,  0.02367407,  0.0236443 ,  0.02362868,
             0.02362312])
 
-    assert_allclose(predicts, predicts_res, atol=1e-6)
+    assert_allclose(predicts, predicts_res, atol=5e-6)
 
 
 def test_ARIMA_exog_predict():

